### PR TITLE
Better website text

### PR DESCRIPTION
**Pull request by LSpenner**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,14 +78,27 @@ <h2 class="text-white-50 mx-auto mt-2 mb-5">Machine Learning for
...
+                    difficult that is.
+                    <br>
+                    <br>
+                    We have built algorithms and machine learning models from all available laboratory data
+                    to help you with these dicisions --
+                    combining your strengths (understanding the reason behind abnormalities) and the strengths of
+                    algorithms (making predictions from big data).
+                    <br>
+                    <br>
+                    Our system makes sure you'll never produce over- or under-strong
```
Comment (SmokinCaterpillar) [inline index.html]: do not use the words always, never etc...

File: index.html [hunk 2]
```diff
@@ -76,14 +78,27 @@ <h2 class="text-white-50 mx-auto mt-2 mb-5">Machine Learning for
...
+                    customer, while at the same time having to deal with fluctuating clinker properties and inconsistent
+                    slag and fly ash qualities. All of these also have different availabilities and prices. Maybe you
+                    even purchase these from different manufacturers, not knowing what exactly to expect. Now you need
+                    to decide recipes and mill targets, while also trying to produce cost optimally. We know how
+                    difficult that is.
+                    <br>
+                    <br>
+                    We have built algorithms and machine learning models from all available laboratory data
+                    to help you with these dicisions --
+                    combining your strengths (understanding the reason behind abnormalities) and the strengths of
```
Comment (SmokinCaterpillar) [inline index.html]: too many round brackets

File: index.html [hunk 3]
```diff
@@ -158,15 +175,20 @@ <h2 class="text-white mb-4">Supporting the Cement Supply Chain
...
             <div class="col-lg-6">
                 <div class="bg-black text-center h-100 project">
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-left">
                             <h4 class="text-white">Leo Spenner</h4>
-                            <p class="mb-0 text-white-50">CEO and smart
-                                guy.</p>
+                            <p class="mb-0 text-white-50">Co-Founder & CEO <br>
+                                Leo has been in close touch with cement all his life, spending many summer holidays
+                                working in a nearby plant run by his family. He holds a M.Sc. in industrial engineering
```
Comment (SmokinCaterpillar) [inline index.html:189]: an M.Sc.

File: index.html [hunk 4]
```diff
@@ -177,15 +199,19 @@ <h4 class="text-white">Leo Spenner</h4>
...
             </div>
             <div class="col-lg-6 order-lg-first">
                 <div class="bg-black text-center h-100 project">
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-right">
                             <h4 class="text-white">Robert Meyer</h4>
-                            <p class="mb-0 text-white-50">CTO and thinks
-                                landing pages are overrated.</p>
+                            <p class="mb-0 text-white-50">Co-Founder & CTO <br>
+                            Robert holds a masters and PhD in Machine Learning (ML) and Neuroinformatics.
```
Comment (SmokinCaterpillar) [inline index.html]: Masters

File: index.html (hunk 2 above)
Comment (LSpenner) [reply index.html]: Please, give me the brackets!!! (they make sense here)

**Pull request merged**